floors/fetcher_test.go

```diff
@@ -396,6 +397,32 @@ func TestValidatePriceFloorRules(t *testing.T) {
 			},
 			wantErr: true,
 		},
+		{
+			name: "Invalid useFetchDataRate",
```

Comment:
Nitpick: Can you update the name here to be `Invalid FetchRate`

Reply:
Done